app/assets/javascripts/client/inventories/InventoryIndexCtrl.js

```diff
@@ -4,15 +4,23 @@
       .controller('InventoryIndexCtrl', InventoryIndexCtrl);
 
   InventoryIndexCtrl.$inject = [
+    '$modal',
     'SessionService',
     'inventory_result'
   ];
 
-  function InventoryIndexCtrl(SessionService, inventory_result) {
+  function InventoryIndexCtrl($modal, SessionService, inventory_result) {
     var vm = this;
 
     vm.user = SessionService.getCurrentUser();
     vm.inventories = inventory_result;
 
+    vm.openInventoryModal = function() {
+      vm.modal = $modal.open({
```

Comment:
I'd love to do this in the future.  I'll put in a task to update all modals we have globally to use this service.